packages/cli/internal/pkg/cli/format/format.go

```diff
@@ -1,8 +1,29 @@
 package format
 
-import "os"
+import (
+	"os"
+	"text/tabwriter"
+)
 
-var Default Formatter = &Text{os.Stdout}
+var Formats = map[string]Formatter{
+	"text":    textWriter,
+	"tabular": tabularWriter,
+}
+var Format string
+
+var textWriter Formatter = &Text{os.Stdout}
+var tabularWriter = &TextTabular{
```

Comment:
We don't have to create all formatters every time we launch AGC.

Reply:
Addressed 👍